src/Extensions/ShareDraftContentSiteTreeExtension.php

```diff
@@ -55,8 +55,14 @@ public function ShareTokenLink()
     {
         $shareToken = $this->getNewShareToken();
 
+        if ($this->owner->SubsiteID) {
```

Comment:
I'm not really a fan of this kind of coupling, but I understand that it's tricky to avoid without having a `silverstripe-sharedraftcontent-subsites` compatibility module, which would also be overkill.

Could you add a class exists check here first just to be safe?

```suggestion
        if (class_exists(Subsite::class) && $this->owner->SubsiteID) {
```

Note: add the `use ...` statement too =)

Reply:
@robbieaverill I had that initially, not entirely sure why I changed it. Adding the class exists check works too.